content/fr/docs/contribute/intermediate.md

```diff
@@ -0,0 +1,834 @@
+---
+title: Contribution intermédiaire
+slug: intermediaire
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+Cette page assume que vous avez lu et compris les tâches dans le topic
+[commencez à contribuer](/docs/contribute/start/) et êtes prêt à
+apprendre différentes manières de contribuer
+
+{{< note >}}
+Certaines tâches nécessitent l'utilisation de Git en ligne de commande ainsi que d'autres outils
+{{< /note >}}
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+Maintenant que vous vous êtes fait la main et que vous avez aidé à la documentation de Kubernetes en
+suivant les principes du topic [commencez à contribuer](/docs/contribute/start/),
+vous êtes surement prêt pour en faire plus. Ces tâches assument que vous avez, ou voulez
+améliorer vos connaissances sur ces différents domaines:
+
+- Les concepts de Kubernetes
+- Kubernetes documentation workflows
+- Where and how to find information about upcoming Kubernetes features
```

Comment:
delete it, this sentence is translated below

Reply:
done